inaugurator/cdrom.py

```diff
@@ -17,7 +18,9 @@ def mount(self):
 
     @contextlib.contextmanager
     def _mount(self, device):
-        if not os.path.isdir(self._MOUNT_POINT):
+        if os.path.exists(self._MOUNT_POINT):
```

Comment:
I think that in that case, something went terribly wrong, and we probably won't want to continue.